cmd/go-qcow2reader-example/read.go

```diff
@@ -67,7 +67,19 @@ func cmdRead(args []string) error {
 
 	buf := make([]byte, bufferSize)
 	sr := io.NewSectionReader(img, offset, length)
-	_, err = io.CopyBuffer(os.Stdout, sr, buf)
+	w := &Writer{os.Stdout}
+
+	_, err = io.CopyBuffer(w, sr, buf)
 
 	return err
 }
+
+// Writer forces os.File to write using our buffer by hiding it's ReadFrom
+// method. io.CopyBuffer is up to XX times faster with this wrrapper.
```

Comment:
Did you want to fulfill `XX`?

Reply:
Maybe just add a link to this PR (https://github.com/lima-vm/go-qcow2reader/pull/42) for the benchmark result and the context

Reply:
I'll link the pr.

Reply:
I updated the docs for the new name.